### ISSUE TITLE
Deprecate the use of 'pt' as measurement for font sizes

### ISSUE THREAD

**Issue by Keimeno** [Satellite-im/Satellite#67]
As can be seen [here](https://github.com/Satellite-im/Satellite/blob/master/src/assets/styles/app.less#L25) `pt` is used as a measurement unit instead of px/em.

This is highly inadvisable since it changes the font size based on its relation to an inch, and not to the screen size of each user. It results in drastically different results between devices ([source](https://css-tricks.com/css-font-size/#pt)).

**Comment by dlum88** [comment]
So just go in all the `.less` files and change out `pt` to `px` or  `em`?

**Comment by RetroPronghorn** [comment]
@dlum88 probably, but we'll need to use some converter to get it as close as possible to the regular size first so it doesn't break the current UI

**Comment by Keimeno** [comment]
A good approach to this would be to first implement visual regression testing with jest and puppeteer, to create snapshots of different pages and or components. This way, after converting all pt to px/em/rem values we can check the difference between each snapshot. This will also help when modifying or creating new components in the future.

**Comment by RetroPronghorn** [comment]
Agreed that would be the perfect solution, however likely once we hit beta we'll be redoing a lot of the UI so a simple pass would be fine for now, I understand the importance of testing however I think it's a bit of work for a small team in PoC phase. 

However if someone were to setup regression testing in a PR I'd be happy to pull it in :)
